app/common/gristUrls.ts

```diff
@@ -995,76 +1005,48 @@ export function isClient() {
   return (typeof window !== 'undefined') && window && window.location && window.location.hostname;
 }
 
+function getCustomizableValue(
+  clientSideConfigKey: keyof GristLoadConfig,
+  serverSideEnvVar: keyof NodeJS.ProcessEnv
+) {
+  return isClient() ? (window as any).gristConfig?.[clientSideConfigKey] : process.env[serverSideEnvVar];
+}
+
 /**
  * Returns a known org "subdomain" if Grist is configured in single-org mode
  * (GRIST_SINGLE_ORG=<org> on the server) or if the page includes an org in gristConfig.
  */
 export function getKnownOrg(): string|null {
-  if (isClient()) {
-    const gristConfig: GristLoadConfig = (window as any).gristConfig;
-    return (gristConfig && gristConfig.singleOrg) || null;
-  } else {
-    return process.env.GRIST_SINGLE_ORG || null;
-  }
+  return getCustomizableValue('singleOrg', 'GRIST_SINGLE_ORG') || null;
 }
```

Comment:
Looks like `getKnownOrg` is (and has been even before this PR) a duplicate of `getSingleOrg`, or do I miss something?

Reply:
Yes it looks like they became identical. Grist used to have some special support for per-site custom domains for which they were distinct.